ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/security-enhancements/cucushift-installer-rehearse-gcp-ipi-security-enhancements-workflow.metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+	"path": "cucushift/installer/rehearse/gcp/ipi/security-enhancements/cucushift-installer-rehearse-gcp-ipi-security-enhancements-workflow.yaml",
```

Comment:
I suggest to use `confidential` and `secureboot` combination keyword to replace `security-enhancement` to easily figure out what features are enabled in the workflow, so I suggest to move the workflow to `cucushift/installer/rehearse/gcp/ipi/secureboot/confidential-computing` or `cucushift/installer/rehearse/gcp/ipi/confidential-computing/secureboot` for the combination.